build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -116,12 +116,12 @@ steps:
     displayName: Run smoke tests (Electron)
     condition: and(succeeded(), eq(variables['VSCODE_STEP_ON_IT'], 'false'))
```

Comment:
Can you change the VSCODE_STEP_ON_IT to RUN_TESTS, and the eq thing should be true.

So we can control all our tests with one variable